sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -13,67 +14,78 @@
 from numpy.testing import assert_array_almost_equal
 from numpy.testing import assert_array_equal
 
-ESTIMATORS = [
-    (label_propagation.LabelPropagation, {"kernel": "rbf"}),
-    (label_propagation.LabelPropagation, {"kernel": "knn", "n_neighbors": 2}),
-    (
-        label_propagation.LabelPropagation,
-        {"kernel": lambda x, y: rbf_kernel(x, y, gamma=20)},
-    ),
-    (label_propagation.LabelSpreading, {"kernel": "rbf"}),
-    (label_propagation.LabelSpreading, {"kernel": "knn", "n_neighbors": 2}),
-    (
-        label_propagation.LabelSpreading,
-        {"kernel": lambda x, y: rbf_kernel(x, y, gamma=20)},
-    ),
+
+ESTIMATOR_CLS = [label_propagation.LabelPropagation, label_propagation.LabelSpreading]
```

Comment:
Actually, since we are always using the same model, I would prefer to use a `pytest.fixture` to generate once for all the different estimators with different parameters.